aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -256,5 +401,48 @@ Tensor& bernoulli_scalar_cuda_(Tensor &self, double p, Generator* gen) {
   return self;
 }
 
+static void uniform_kernel_cuda(TensorIterator& iter, double from_, double to_, Generator* gen) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "uniform_cuda", [&] {
+    auto from = static_cast<scalar_t>(from_);
+    auto to = static_cast<scalar_t>(to_);
+    AT_CHECK(from <= to,
+      "uniform_ expects to return a [from, to) range, but found from=", from,
+      " > to=", to);
+    AT_CHECK((to - from) <= std::numeric_limits<scalar_t>::max(),
+          "uniform_ expects to-from ≤ std::numeric_limits<double>::max(), but found to=", to,
```

Comment:
Use of Unicode in source file makes me unhappy

Reply:
Didn't quite get it. So just change the message to `uniform_ expects (to-from) difference to be less than or equal to the maximum finite value representable by the numeric type scalar_t`? :)

Reply:
oh `≤` I see

Reply:
I totally copy-pasted the requirements from here: https://en.cppreference.com/w/cpp/numeric/random/uniform_real_distribution/uniform_real_distribution

Reply:
Also, the error message always says `<double>` but the code may operate on float and half.